### DISCUSSION TITLE
Slack notifications missing sender Amazfit Bip 3 Pro

### DISCUSSION THREAD

**Discussion by Moshikol** [zepp-health/zeppos-forum#380] [User Feedback]
Hi, im using the Zepp application version 8.13.1 on my Galaxy s24 Ultra, when im getting a new message from the Slack application I only see the message content and not the sender/ channel name see picture:
![20241008_194212](https://github.com/user-attachments/assets/8f20e6ab-51b1-4712-95cd-fd47f4243510)

As you can see, the message content is "test" and no sender is shown.
Any idea how to fix it? Thanks!

**Comment by MyoungXUE** [comment]
Hello, from the picture, it seems that your device is Amazfit Bip 3, is that right?

Did Slack notifications have channels and senders before, or did this issue occur after upgrading to 8.13.1 version of Zepp?

I suggest upgrading Zepp to the latest version first and then observing if this problem occurs.

**Reply by Moshikol** [reply]
Yes, its the Bip 3 pro, it never had it before, and im using the latest version available now its 9.0.5 still no change

**Comment by Moshikol** [comment]
Any news?

**Comment by Moshikol** [comment]
@MyoungXUE any idea  how to fix it? Or can you maybe direct me to the open source repo that controls it (if it's an open source of course)
Thanks!
